scripts/import/import_variant_synonyms

```diff
@@ -45,50 +45,83 @@ use Bio::EnsEMBL::Variation::Source;
 
 our $DEBUG = 0;
 
-my ($data_file, $registry_file, $species, $source_name, $clean, $source_version, $source_url, $source_description);
+my ($data_file, $registry_file, $species, $source_name, $clean, $source_version, $source_url, $source_description, $host,
+   $port, $user, $pass, $db_name);
 
 GetOptions ("data_file=s"          => \$data_file,
             "species=s"            => \$species,
             "source_name=s"        => \$source_name,
             "source_url=s"         => \$source_url,
             "source_version=s"     => \$source_version,
             "source_description=s" => \$source_description,
-            "registry=s"           => \$registry_file,
+            "registry=s"           => \$registry_file,            
+            "host=s"               => \$host,
+            "port=s"               => \$port,
+            "user=s"               => \$user,
+            "pass=s"               => \$pass,
+            "db_name=s"            => \$db_name,
             "clean"                => \$clean 
             );
 
-usage() unless defined $registry_file && defined $source_name;
+usage() unless defined $registry_file && defined $source_name; 
 
 $species ||= 'homo_sapiens';
 
 
 my $reg = 'Bio::EnsEMBL::Registry';
 $reg->load_all($registry_file);
-my $dba = $reg->get_DBAdaptor($species, 'variation');
+my $dba = $reg->get_DBAdaptor($species, 'variation'); 
+
 ## include failed variants to avoid missing any links
 $dba->include_failed_variations(1);
 
 
-my $source = get_source($species, $dba, $source_name, $source_version, $source_url, $source_description );
-
-my $synonyms;
-
 ## collect synonyms by source
 if($source_name eq 'PharmGKB'){
+  my $source = get_source($species, $dba, $source_name, $source_version, $source_url, $source_description );
+
+  my $synonyms;
 
   ## collect data files if not already available
   $data_file = download_PharmGKB() unless $data_file;
 
   ## extract synonyms from file
   $synonyms  = extract_PharmGKB($data_file);
+  
+  ## add synonyms to the database
+  import_synonyms($synonyms, $source, $dba, $species); 
 }
-else{
-    die "data source : $source_name not supported\n";
-}
 
+elsif($source_name eq 'dbSNP'){
+  ### import synonyms from production database ### Access directly database because this database doesn't have adaptors 
+  my $prod_dbc = Bio::EnsEMBL::DBSQL::DBConnection->new
+    ( '-host'    => $host,
+      '-port'    => $port, 
+      '-user'    => $user,
+      '-pass'    => $pass,
+      '-dbname'  => $db_name, 
+      '-driver'  =>  'mysql' 
+    ); 
+
+  my $synonyms_output = query_synonyms_data($prod_dbc, '*', 'human_synonym'); 
```

Comment:
Could you specify the columns here to ensure they are returned in the expected order?